### PR TITLE
Change remark to description and update sameperson check

### PR DESCRIPTION
**Pull request by isanidiot**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/UpdateCommand.java
Comment (ReaganTan00) [inline src/main/java/seedu/address/logic/commands/UpdateCommand.java:135]: I think we should enhance UpdateCommand to allow updating of a client's description.
Can implement as a separate PR/Issue

Comment (ReaganTan00) [reply src/main/java/seedu/address/logic/commands/UpdateCommand.java:135]: Current implementation does not allow updating of descriptions

File: src/main/java/seedu/address/model/person/Person.java
Comment (ReaganTan00) [inline src/main/java/seedu/address/model/person/Person.java:134]: Updated samePerson check 👍

File: src/test/java/seedu/address/model/person/PersonTest.java
Comment (ReaganTan00) [inline src/test/java/seedu/address/model/person/PersonTest.java:51]: Updated test due to changed isSamePerson check 👍

**Review by ReaganTan00 — COMMENTED** [review]
LGTM after resolving merge conflicts
To consider:
Allowing updating of description through UpdateCommand

**Review by ReaganTan00 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/UpdateCommand.java
Comment (isanidiot) [reply src/main/java/seedu/address/logic/commands/UpdateCommand.java:136]: right now editting description has its own command

**Pull request merged**
